src/vitepress/styles/index.css

```diff
@@ -1,3 +1,3 @@
 @import '../../core/styles/index.css';
-@import './fonts.css';
+@import 'vitepress/dist/client/theme-default/styles/fonts.css';
```

Comment:
A non-blocking small suggestion, do you think this line of code better be moved to the top as a "3rd-party" import?